src/app/components/article/ArticleForm.js

```diff
@@ -0,0 +1,402 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Slideout from '../cds/slideout/Slideout';
+import ButtonMain from '../cds/buttons-checkboxes-chips/ButtonMain';
+import TagList from '../cds/menus-lists-dialogs/TagList';
+import TextArea from '../cds/inputs/TextArea';
+import TextField from '../cds/inputs/TextField';
+import LanguagePickerSelect from '../cds/inputs/LanguagePickerSelect';
+import LimitedTextArea from '../layout/inputs/LimitedTextArea';
+import DeleteIcon from '../../icons/delete.svg';
+import inputStyles from '../../styles/css/inputs.module.css';
+import { safelyParseJSON, truncateLength } from '../../helpers';
+import styles from './ArticleForm.module.css';
+
+const ArticleForm = ({
+  handleSave,
+  onClose,
+  handleBlur,
+  handleDelete,
+  articleType,
+  mode,
+  article,
+  team,
+  projectMedia,
+}) => {
+  const title = (
+    <>
+      {articleType === 'explainer' && mode === 'create' && <FormattedMessage id="article-form-explainer-create-title" defaultMessage="Create Explainer" description="Title for the slideout create explainer form" />}
+      {articleType === 'explainer' && mode === 'edit' && <FormattedMessage id="article-form-explainer-edit-title" defaultMessage="Edit Explainer" description="Title for the slideout edit explainer form" />}
+      {articleType === 'fact check' && mode === 'create' && <FormattedMessage id="article-form-fact-check-create-title" defaultMessage="Create New Claim & Fact-Check" description="Title for the slideout create fact check form" />}
+      {articleType === 'fact check' && mode === 'edit' && <FormattedMessage id="article-form-fact-check-edit-title" defaultMessage="Edit Claim & Fact-Check" description="Title for the slideout edit fact check form" />}
+    </>
+  );
+
+  const [claimDescription, setClaimDescription] = React.useState(articleType === 'fact check' && projectMedia?.suggested_main_item ? projectMedia.suggested_main_item.claim_description : '');
+  const [claimContext, setClaimContext] = React.useState('');
+
+  const languages = safelyParseJSON(team.get_languages) || ['en'];
+  const defaultArticleLanguage = languages && languages.length === 1 ? languages[0] : null;
+  const [articleTitle, setArticleTitle] = React.useState((article && article.title) ? article.title : '');
+  const [summary, setSummary] = React.useState(article && article.summary ? article.summary : '');
+  const [url, setUrl] = React.useState((article && article.url) ? article.url : '');
```

Comment:
Suggestion: I think these default states could be more concise and easier to read: `article?.prop || ''`